.travis.yml

```diff
@@ -1,19 +1,24 @@
 language: python
 
 python:
-  - 2.7
+  - 3.7
 
 services:
   - docker
 
 install:
+  - pip install coverage coveralls
   - docker run --name avalon-mongo -d mongo
-  - docker build -t avalon/core -f Dockerfile-maya2016 .
+  - . build_docker.sh
 
 script:
   - . test_docker.sh
   - . build_docs.sh
 
+after_success:
+  - coverage combine
+  - coveralls
```

Comment:
Seems to be issues with this command; https://travis-ci.org/github/getavalon/core/builds/712117798#L608

Reply:
Please see comment https://github.com/getavalon/core/pull/559#issuecomment-664370166